### ISSUE TITLE
Saving HTML files causes a hot reload

### ISSUE THREAD

**Issue by jagdishjadeja** [Dart-Code/Dart-Code#994]
when plugin is running and i open some html, js or any other file (which is not related to flutter app or dart or anything still app gets reloaded ) then make some modification in that html, js or any other file and save it my app gets reloaded.
Where my app is only supposed to be reloaded when an app related file is modified.

**About VS Code :**
Version 1.23.1
Commit d0182c3417d225529c6d5ad24b7572815d0de9ac
Date 2018-05-10T17:11:17.614Z
Shell 1.7.12
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64

**Dart code version** :  2.13.2

**Flutter details:**
Flutter 0.4.4 • channel beta • https://github.com/flutter/flutter.git
Framework • revision f9bb4289e9 (4 weeks ago) • 2018-05-11 21:44:54 -0700
Engine • revision 06afdfe54e
Tools • Dart 2.0.0-dev.54.0.flutter-46ab040e58

**Comment by DanTup** [comment]
> then make some modification in that html, js or any other file and save it my app gets reloaded.

Can you confirm you're seeing this for js files and not just HTML? I can see how it can happen for HTML (because we consider that "analyzable") but if it's happening for JS files, there's something else wrong.

**Comment by jagdishjadeja** [comment]
@DanTup yes it reloads even when i save js file

**Comment by DanTup** [comment]
@jagdishjadeja Can you enable `Flutter Run Logging`(https://dartcode.org/docs/logging/#flutter-run) and reproduce and attach the log, and also check in the status bar when you're editing your JS file that it correctly says "JavaScript" as the language (or, attach a screenshot including the status bar).

**Comment by DanTup** [comment]
Aha, I figured out what's going on! :)

As part of #955 I fixed that saving *any file* would trigger a hot reload, but that hadn't shipped yet. When investigating your issue, I discovered that even with that fix, we'd still trigger on HTML files.

So, #955 fixes it for most file types, and the new fix on this issue fixes it for HTML too. Both fivxes are in v2.14.0 Beta 2 which will should be available later today.
